proxies/go/handler_client.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"encoding/json"
+	"github.com/devcyclehq/go-server-sdk"
+	"net/http"
+)
+
+type clientRequestBody struct {
+    ClientId         string   `json:"clientId"`
+    SdkKey      string   `json:"sdkKey, omitempty"`
+    EnableCloudBucketing bool `json:"enableCloudBucketing"`
+    WaitForInitialization bool `json:"waitForInitialization"`
+    Options devcycle.DVCOptions `json:"options"`
+}
+
+type clientResponseBody struct {
+    Message         string   `json:"message, omitempty"`
+    AsyncError      string   `json:"asyncError, omitempty"`
+    Exception       string   `json:"exception, omitempty"`
+}
+func clientHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think you're missing exception handling

Reply:
like for the case where the env key is missing we'd want to return `exception: Missing environment key....`

Reply:
ah I think I actually missed a step entirely in the initialization process. The exception handling is going to look real strange for this endpoint because theres no way for `waitForInitialization: false` to work with the sdk right now

Reply:
oh true, I guess there's no way to await initialization for the sdk right now

Reply:
but we still need to wrap the whole `NewDVCClient` call in a try/catch and return `exception` in the catch

Reply:
yeah, adding that now along with actually initializing local bucketing properly